lua/nvim-tree/node.lua

```diff
@@ -21,6 +21,7 @@
 ---@field group_next Node|nil
 ---@field nodes Node[]
 ---@field open boolean
+---@field hidden_count table -- Each field of this table is a key for source and value for count
```

Comment:
Could we please rename this to `hidden_stats` ?

I can see myself thinking it's a number...

Reply:
Will do